src/App.jsx

```diff
@@ -228,6 +239,100 @@ function App() {
         )}
       </div>
 
+      <Dialog
+        sx={{ borderRadius: "100px" }}
+        open={openShareModal}
+        onClose={() => setOpenShareModal(false)}
+      >
+        <div
+          style={{
+            backgroundColor: "var(--bg-color)",
+            padding: "20px",
+            textAlign: "center",
+            color: "var(--color)",
+            border: "2px solid var(--color)",
+          }}
+        >
+          <img
+            src="https://user-images.githubusercontent.com/27727921/185767526-a002a17d-c12e-4a6a-82a4-dd1a13a5ecda.png"
```

Comment:
try the logo from assets/logo.png , fix alt to be dummygram, also the classname should modal signup img